sub-logs/terabox-dl.json

```diff
@@ -0,0 +1,15 @@
+{
+    "owner": {
+        "username": "jinix6",
+        "email": "lipanghadai@gmail.com"
+    },
+
+    "target": {
+        "CNAME": {
+            "name": "terabox-dl",
+            "value": "jinix6.github.io/terabox-downloader"
```

Comment:
Invalid CNAME value, should be a hostname.

E.g. `dibster.org`

Not `https://dibster.org/about`